core/src/main/java/io/grpc/transport/ServerTransportListener.java

```diff
@@ -48,4 +49,9 @@
    */
   ServerStreamListener streamCreated(ServerStream stream, String method,
       Metadata.Headers headers);
+
+  /**
+   * The transport completed shutting down. All resources have been released.
+   */
+  void transportTerminated();
```

Comment:
No, I wasn't trying to make that distinction. I was instead trying to make the distinction that ```shutdown``` is for initiation and ```terminated``` is for completion. For example, I wouldn't find it weird if one employee told another (who may be currently working), "We've been shutdown." Even though not everyone has been notified and some working is still wrapping up, I wouldn't find it strange for someone to say they _were_ shutdown (past tense) even though they are shutting down now.